ur

```diff
@@ -1 +1 @@
-Subproject commit 4fffe07b27706f5c6c0bfb007b42a27687566380
+Subproject commit ee4eea5613fa097693b6474d1c427770a4a8fd6d
```

Comment:
Hmm, I'm not sure what the cleanest way is - you could revert the commit they went in on and then make a new commit only with the files you want, then push both the revert commit and the new commit?  I am guessing this will drop you into merge conflict hell though.